recipes/pymol-open-source/meta.yaml

```
@@ -0,0 +1,74 @@
+{% set version = "2.4.0" %}
+{% set number = "0" %}
+{% set sha256 = "5ede4ce2e8f53713c5ee64f5905b2d29bf01e4391da7e536ce8909d6b9116581" %}
+
+package:
+  name: pymol-open-source
+  version: {{ version }}
+
+source:
+  url: https://github.com/schrodinger/pymol-open-source/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: {{ number }}
+  script: "{{ PYTHON }} setup.py install --use-msgpackc=no --no-libxml --no-vmd-plugins"
+  skip: True  # [py < 37]
+  skip: True  # [osx]
+
+requirements:
+  build:
+    - python
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - mesalib                        # [linux]
+    - libglu                         # [linux]
+    - libpng
+    - freetype
+    - pyqt
+    - glm
```

Most of these aren't needed in build:
```suggestion
    - {{ compiler('c') }}
    - {{ compiler('cxx') }}
```

---

You are right, I tested it without most of these packages and it still works fine. Therefore I removed them. However, Python is required in ``{{ PYTHON }} setup.py install``, am I right?

---

Python should be in `host`

---

You are right: I removed Python from the build requirements and it still works. Thank you for your help!